### PR TITLE
Add a bunch of new trainers, some new attacks and some new lands

### PR DESCRIPTION
**Pull request by brooks42**

### CODE REVIEW

File: ptcg_cockatrice.xml [hunk 1]
```diff
@@ -1124,9 +1124,69 @@
...
+			<name>Lorelei</name>
+			<set picURL="https://i.imgur.com/WMmQcyD.jpeg" rarity="mythic">PTCG</set>
+			<color>WU</color>
+			<manacost>1{W/U}{W/U}</manacost>
+			<tablerow>1</tablerow>
+			<type>Legendary Trainer - Lorelei</type>
+			<loyalty>2</loyalty>
+			<text>[+1]: Target player gains 3 life.
+				[0]: Until end of turn, Water and Ice Pokémon you control get +0/+2.
+				[-5]: Put any number of Water and Ice Pokémon from your hand onto the battlefield.</text>
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:1152]:
```suggestion
				[-5]: Put any number of Water and Ice Pokémon cards from your hand onto the battlefield.</text>
```

File: ptcg_cockatrice.xml [hunk 2]
```diff
@@ -1124,9 +1124,69 @@
...
+		</card>
+		<card>
+			<name>Agatha</name>
+			<set picURL="https://i.imgur.com/WMmQcyD.jpeg" rarity="mythic">PTCG</set>
+			<color>BU</color>
+			<manacost>3{B/U}{B/U}</manacost>
+			<tablerow>1</tablerow>
+			<type>Legendary Trainer - Agatha</type>
+			<loyalty>3</loyalty>
+			<text>[+1]: Exile the top 3 cards of target player's library. You can play those cards until end of turn.
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:1174]:
```suggestion
			<text>[+1]: Exile the top three cards of target player's library. You may play those cards until end of turn.
```

File: ptcg_cockatrice.xml [hunk 3]
```diff
@@ -1124,9 +1124,69 @@
...
+		</card>
+		<card>
+			<name>Lance</name>
+			<set picURL="https://i.imgur.com/WMmQcyD.jpeg" rarity="mythic">PTCG</set>
+			<color>WRG</color>
+			<manacost>4WRG</manacost>
+			<tablerow>1</tablerow>
+			<type>Legendary Trainer - Lance</type>
+			<loyalty>4</loyalty>
+			<text>[+2]: Deal 3 damage to target Pokémon. You gain 3 life.
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:1186]:
```suggestion
			<text>[+2]: Lance deals 3 damage to target Pokémon. You gain 3 life.
```

File: ptcg_cockatrice.xml [hunk 4]
```diff
@@ -1245,6 +1305,24 @@
...
 			<text>Remove all damage from target Pokémon.</text>
 		</card>
+		<card>
+			<name>Super Potion</name>
+			<set picURL="https://images.pokemontcg.io/ecard1/156_hires.png" rarity="common">PTCG</set>
+			<color>W</color>
+			<manacost>1WW</manacost>
+			<type>Instant - Medicine</type>
+			<tablerow>3</tablerow>
+			<text>Remove all damage from target Pokémon. It gains indestructible until end of turn.</text>
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:1315]:
```suggestion
			<text>Target Pokémon gains indestructible until end of turn.</text>
```

File: ptcg_cockatrice.xml [hunk 5]
```diff
@@ -1245,6 +1305,24 @@
...
+		</card>
+		<card>
+			<name>Lemonade</name>
+			<set picURL="https://images.pokemontcg.io/ecard1/156_hires.png" rarity="common">PTCG</set>
+			<color></color>
+			<manacost>5</manacost>
+			<type>Instant - Medicine</type>
+			<tablerow>3</tablerow>
+			<text>Remove all damage from target Pokémon. If you do, untap it and it gains first strike until end of turn.</text>
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:1324]:
```suggestion
			<text>Remove all damage from target Pokémon. Untap it and it gains first strike until end of turn.</text>
```

File: ptcg_cockatrice.xml [hunk 6]
```diff
@@ -1263,6 +1341,42 @@
...
+			<text>Target Pokémon gains hexproof until end of turn.</text>
+		</card>
+		<card>
+			<name>Antidote</name>
+			<set picURL="https://images.pokemontcg.io/base1/84_hires.png" rarity="common">PTCG</set>
+			<color></color>
+			<manacost>1W</manacost>
+			<type>Sorcery - Medicine</type>
+			<tablerow>3</tablerow>
+			<text>Remove up to 3 -1/-1 counters from target Pokémon, or up to 3 poison counters from target player.</text>
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:1378]:
```suggestion
			<text>Remove up to three -1/-1 counters from target Pokémon or up to three poison counters from target player.</text>
```

File: ptcg_cockatrice.xml [hunk 7]
```diff
@@ -1390,6 +1522,25 @@
...
 			<text>{T}: Search your library for a Water Pokémon with an evolve ability, then shuffle and put that card third from the top.</text>
 		</card>
+		<card>
+			<name>Ancient Slab</name>
+			<set picURL="https://images.pokemontcg.io/sm12/197_hires.png" rarity="common">PTCG</set>
+			<color></color>
+			<manacost>2</manacost>
+			<type>Artifact</type>
+			<tablerow>1</tablerow>
+			<text>{1}, {T}: add one mana of any color.</text>
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:1532]:
```suggestion
			<text>{1}, {T}: Add one mana of any color.</text>
```

File: ptcg_cockatrice.xml [hunk 8]
```diff
@@ -1541,6 +1692,36 @@
...
 		</card>
+		<card>
+			<name>Mega Drain</name>
+			<set picURL="https://images.pokemontcg.io/ex3/90_hires.png" rarity="uncommon">PTCG</set>
+			<color>G</color>
+			<manacost>G</manacost>
+			<type>Enchantment - TM</type>
+			<tablerow>1</tablerow>
+			<text>Enchant Grass or Bug Pokémon
+				Whenever enchanted Pokémon deals combat damage, you gain half that much life rounded up.</text>
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:1703]:
```suggestion
				Whenever enchanted Pokémon deals combat damage, you gain half that much life, rounded up.</text>
```

File: ptcg_cockatrice.xml [hunk 9]
```diff
@@ -1541,6 +1692,36 @@
...
+		</card>
+		<card>
+			<name>Pay Day</name>
+			<set picURL="https://images.pokemontcg.io/ex3/90_hires.png" rarity="uncommon">PTCG</set>
+			<color></color>
+			<manacost>3</manacost>
+			<type>Enchantment - TM</type>
+			<tablerow>1</tablerow>
+			<text>Enchant Pokémon
+				Whenever enchanted Pokémon deals combat damage, create a Treasure token. (It's an artifact with "{T}, Sacrifice this artifact: Add one mana of any color.")</text>
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:1723]:
```suggestion
				Whenever enchanted Pokémon deals combat damage, create a Treasure token.</text>
```

File: ptcg_cockatrice.xml [hunk 10]
```diff
@@ -1646,5 +1827,24 @@
...
 		</card>
+		<card>
+			<name>Cinnabar Lab</name>
+			<set picURL="https://i.imgur.com/HEb2JN5.jpeg" rarity="rare">PTCG</set>
+			<color></color>
+			<manacost></manacost>
+			<type>Land</type>
+			<tablerow>0</tablerow>
+			<text>{T}: Add {1}
+				{T}: Add {3}. This mana can only be used to pay for fossil abilities.</text>
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:1838]:
```suggestion
				{T}: Add {3}. Spend this mana only to activate fossil abilities.</text>
```

File: ptcg_cockatrice.xml [hunk 11]
```diff
@@ -1646,5 +1827,24 @@
...
+		</card>
+		<card>
+			<name>Safari Zone</name>
+			<set picURL="https://i.imgur.com/HEb2JN5.jpeg" rarity="uncommon">PTCG</set>
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:1842]:
```suggestion
			<set picURL="https://i.imgur.com/HEb2JN5.jpeg" rarity="common">PTCG</set>
```

**Review by triggrhappy0 — CHANGES_REQUESTED** [review]

File: ptcg_cockatrice.xml [hunk 12]
```diff
@@ -1124,9 +1124,69 @@
...
+		<card>
+			<name>Lorelei</name>
+			<set picURL="https://i.imgur.com/WMmQcyD.jpeg" rarity="mythic">PTCG</set>
+			<color>WU</color>
+			<manacost>1{W/U}{W/U}</manacost>
+			<tablerow>1</tablerow>
+			<type>Legendary Trainer - Lorelei</type>
+			<loyalty>2</loyalty>
+			<text>[+1]: Target player gains 3 life.
+				[0]: Until end of turn, Water and Ice Pokémon you control get +0/+2.
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:1151]:
```suggestion
				[0]: Water and Ice Pokémon you control get +0/+2 until end of turn.
```

File: ptcg_cockatrice.xml [hunk 13]
```diff
@@ -1124,9 +1124,69 @@
...
+		<card>
+			<name>Lance</name>
+			<set picURL="https://i.imgur.com/WMmQcyD.jpeg" rarity="mythic">PTCG</set>
+			<color>WRG</color>
+			<manacost>4WRG</manacost>
+			<tablerow>1</tablerow>
+			<type>Legendary Trainer - Lance</type>
+			<loyalty>4</loyalty>
+			<text>[+2]: Deal 3 damage to target Pokémon. You gain 3 life.
+				[0]: Until end of turn, Lance becomes a 6/6 Human creature with indestructible and flying that's still a Trainer. Prevent all damage that would be dealt to him this turn.
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:1187]:
```suggestion
				[0]: Until end of turn, Lance becomes a 6/6 Dragon Pokémon with flying that's still a Trainer. Prevent all damage that would be dealt to him this turn.
```

File: ptcg_cockatrice.xml [hunk 14]
```diff
@@ -1390,6 +1522,25 @@
...
+			<name>Ancient Slab</name>
+			<set picURL="https://images.pokemontcg.io/sm12/197_hires.png" rarity="common">PTCG</set>
+			<color></color>
+			<manacost>2</manacost>
+			<type>Artifact</type>
+			<tablerow>1</tablerow>
+			<text>{1}, {T}: add one mana of any color.</text>
+		</card>
+		<card>
+			<name>Bill's Computer</name>
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:1535]:
```suggestion
			<name>Pokémon Storage System</name>
```

File: ptcg_cockatrice.xml [hunk 15]
```diff
@@ -1390,6 +1522,25 @@
...
+			<set picURL="https://images.pokemontcg.io/sm12/197_hires.png" rarity="common">PTCG</set>
+			<color></color>
+			<manacost>4</manacost>
+			<type>Artifact</type>
+			<tablerow>1</tablerow>
+			<text>At the beginning of each opponent's upkeep, exile all cards from your hand face down.
+At the beginning of your upkeep, return all cards you own exiled with Bill's Computer to your hand, then draw a card.</text>
```
Comment (triggrhappy0) [inline ptcg_cockatrice.xml:1542]:
```suggestion
At the beginning of your upkeep, return all cards you own exiled with Pokémon Storage System to your hand, then draw a card.</text>
```

File: ptcg_cockatrice.xml (hunk 10 above)
Comment (brooks42) [reply ptcg_cockatrice.xml:1838]: Thanks, wasn't quite sure how to word this and couldn't find any good examples in Cockatrice

**Pull request merged**